kornia/augmentation/_2d/intensity/random_snow.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Dict, Optional
+
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.color import hls_to_rgb, rgb_to_hls
+from kornia.core import Tensor
+
+
+class RandomSnow(IntensityAugmentationBase2D):
+    r"""Generates snow effect on given tensor image or a batch tensor images.
+
+    Args:
+        snow_coefficient (float): A float between 0 and 1 that controls the amount of snow to add to the
```

Comment:
You do not need to type it in the docs. It will be done automatically. So does the default values.